packages/node/test/integration/utils.ts

```diff
@@ -95,9 +95,9 @@ export function makeProposalRequest(
     asset: {
       assetType: AssetType.ETH
     },
-    myDeposit: ethers.utils.bigNumberify("1"),
-    peerDeposit: ethers.utils.bigNumberify("1"),
-    timeout: ethers.utils.bigNumberify("1"),
+    myDeposit: One,
```

Comment:
@snario I'm not particularly sure about this one. I think it's clearer to use `bigNumerify("1")`, but that's just me, take it or leave it :P